### ISSUE TITLE
Empty many to many relationships cause an exception

### ISSUE THREAD

**Issue by johncarter-** [ryangjchandler/orbit#116]
When accessing empty model many to many relationships you get a `SQLSTATE[HY000]: General error: 1 no such table: the_pivot_table_name` exception.

**How to replicate**

1. Clone https://github.com/johncarter-/laravel-orbit-many-to-many

2. Switch to the `issue/many-to-many-no-such-table` branch.

3. Run  `php artisan test --filter it_can_handle_empty_many_to_many_relationships tests/Feature/OrbitalManyToManyTest.php`

**Comment by johncarter-** [comment]
If you create then delete a pivot model to sort of "initialise" the "table"/directory it works. See the follow up test in that branch. I took a bit of source dive 🤿, but after a while trying to figure it out I didn't know my arse from my elbow. 🤷‍♂️

**Comment by ryangjchandler** [comment]
@johncarter- Yeah, the codebase is a bit messy right now. My 2.x branch is much cleaner, I think.

**Comment by johncarter-** [comment]
Oh I didn't mean there was anything wrong with your code 😯 I just meant I was in over my head.

I can use the create and delete workaround for now, no pressure at all👌

**Comment by ryangjchandler** [comment]
@johncarter- Sweet!
